tlair/src/test/scala/at/forsyte/apalache/tla/typecomp/TestFunBuilder.scala

```diff
@@ -47,11 +47,6 @@ class TestFunBuilder extends BuilderTest {
         )
     )
 
-    // test fail on n = 0
-    assertThrows[IllegalArgumentException] {
-      build(builder.recMixed())
-    }
-
```

Comment:
Similar to the comment above, it would be great to keep the test and assert that it does not fail anymore.